CesiumGeospatial/include/CesiumGeospatial/GlobeRectangle.h

```diff
@@ -259,6 +261,17 @@ class CESIUMGEOSPATIAL_API GlobeRectangle final {
    */
   GlobeRectangle computeUnion(const GlobeRectangle& other) const noexcept;
 
+  /**
+   * @brief Computes the normalized position of the given cartographic
+   * coordinate on this globe rectangle, with (0.0, 0.0) representing southwest
+   * and (1.0, 1.0) representing northeast.
```

Comment:
It's worth mentioning that it interpolates linearly in longitude/latitude space. Not in ECEF or some map projection. Probably borderline obvious, but still.